src/libzippp.h

```diff
@@ -538,6 +547,8 @@ namespace libzippp {
         
         void** bufferData;
         libzippp_uint64 bufferLength;
+
+        ErrorHandlerCallback errorHandlingCallback = nullptr;
```

Comment:
Initialize the variable in the constructor initializer.

Reply:
done